docs/development/translations.rst

```diff
@@ -22,6 +28,13 @@ about anything in this document, here's how to ask for help:
 
 * Post a message in the `translations category of the SecureDrop forum`_
 * Chat in the `SecureDrop instant messaging channel`_
+
+   * Localization Lab, with whom we :ref:`coordinate SecureDrop's
```

Comment:
A link to https://www.localizationlab.org/ would be better here than later on in the doc when we're getting into the weeds of when translations happen during code changes and such.

Reply:
Added in 6ac2ad8d6c26c0dd6edd7e69cfe066bef85fa588!